core/tests/ts-integration/tests/api/snapshots-creator.test.ts

```diff
@@ -57,18 +57,19 @@ describe('Snapshots API tests', () => {
         const fullSnapshot = await getSnapshot(l1BatchNumber);
         const miniblockNumber = fullSnapshot.miniblockNumber;
 
+        const protoPath = path.join(process.env.ZKSYNC_HOME as string, 'core/lib/types/src/proto/mod.proto');
```

Comment:
Just something to keep in mind: This would break if the Protobuf declarations were more complex (e.g., depended on other modules).

Reply:
agreed